jdaviz/configs/mosviz/tests/test_parsers.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from zipfile import ZipFile
+import pathlib
+
+from astropy.utils.data import download_file
+
+
+@pytest.mark.remote_data
+def test_niriss_parser(mosviz_app, tmpdir):
+
+    test_data = 'https://stsci.box.com/shared/static/9lkf5zha6zkf8ujnairy6krobbh038wt.zip'
+    fn = download_file(test_data, cache=True)
+    with ZipFile(fn, 'r') as sample_data_zip:
+        sample_data_zip.extractall(tmpdir)
+
+    level3_path = (pathlib.Path(tmpdir) / 'NIRISS_for_parser_p0171')
+
+    data_dir = level3_path
+
+    mosviz_app.load_niriss_data(data_dir)
+
+    assert len(mosviz_app.app.data_collection) == 80
```

Comment:
Still don't know why we need 80 data entries to do this one test but if you said it is 100 MB, I'll let it go until CI complains...

Reply:
Mainly it's the NIRISS dataset I have available, and I'm busy enough right now that I didn't want to go through the files to cut out the right HDUs that correspond to each other or make a dummy dataset with all the data types needed.